script.js

```diff
@@ -22,3 +22,223 @@ navLinksEl.forEach((item) => {
+    FeaturedImage: 'images/Snapshoot Portfolio (3).png',
+  },
+];
+
+const sectionHide = document.querySelector('.container');
+const btnEl1 = document.querySelector('.btn-one');
+const btnEl2 = document.querySelector('.btn-two');
+const btnEl3 = document.querySelector('.btn-three');
+const btnEl4 = document.querySelector('.btn-four');
+const body = document.querySelector('body');
+
+btnEl1.addEventListener('click', () => {
+  sectionHide.classList.toggle('hide-container');
+  body.classList.toggle('background-img');
+  const Popsection = document.createElement('section');
+  Popsection.className = 'Popup-Section';
+  Popsection.innerHTML = `
+      <div class="icon-holder">
+      <h2>${workCards[0].Name} </h2>
+      <a onclick="showMenu(close)"><i class="fa-solid fa-xmark close-cards"></i></a>
+      </div>
+      <article class="wrap-frame">
+      <p class="frame canopy">CANOPY</p>
+      <img src="images/Counter.png" alt="">
+      <p class="frame">Back End Dev</p>
+      <img class="frame" src="images/Counter.png" alt="" class="card-img">
+      <p class="frame">2015</p>
+      </article>
+      <img src= "${workCards[0].FeaturedImage}" alt="Snapshoot" class="card-img desk-img"> 
+      <div class="right-pop">
+      <p class="par pop-par">${workCards[0].Description} </p>
+      <div class="column-pop">
+      <div class="lang-button">
+      <ul>
+          <li><a href="#">${workCards[0].Technologies[0]}</a></li>
+          <li> <a href="#">${workCards[0].Technologies[1]}</a></li>
+          <li><a href="#">${workCards[0].Technologies[2]}</a></li>
+      </ul>
+      <ul class="Desktop-vers">
+          <li><a href="#">${workCards[0].Technologies[3]}</a></li>
+          <li> <a href="#">${workCards[0].Technologies[4]}</a></li>
+          <li><a href="#">${workCards[0].Technologies[5]}</a></li>
+      </ul>
+  </div>
+  <div class="popBtn">
+  <a href="https://saluumaa.github.io/Personal-portfolio/"><button type="button" class="pop-btn">${workCards[0].LiveVersion}  <i class="fa-solid fa-power-off in-btn"></i></button></a>
+  <a href="https://github.com/saluumaa/Saluma-s-Portfolio"><button type="button" class="pop-btn">${workCards[0].source} <i class="fa-brands fa-github in-btn"></i> </button></a>
```

Comment:
- [ ] Kindy try to decrease the padding of the buttons here to be like the Figma Design 

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/216100830-ee54e032-d9fa-447d-99ef-eacb410eec92.png)| ![image](https://user-images.githubusercontent.com/83360636/216100718-9d0e3a71-ae64-4c56-b75a-5bf6fc5c3169.png)|